kube-core/src/params.rs

```diff
@@ -429,6 +451,28 @@ impl WatchParams {
         self
     }
 
+    /// Configure typed label selectors
+    ///
+    /// Configure typed selectors from [`Selector`](crate::Selector) and [`Expression`](crate::Expression) lists.
+    ///
+    /// ```
+    /// use kube::api::WatchParams;
+    /// use kube_core::{Expression, Selector, ParseExpressionError};
+    /// use k8s_openapi::apimachinery::pkg::apis::meta::v1::LabelSelector;
+    /// let selector: Selector = Expression::In("env".into(), ["development".into(), "sandbox".into()].into()).into();
```

Comment:
Note to future self; this heavy use of `into` when using local str slices could be reduced in the future with some macro magic ala:

```rust
selector::in! { "env", ("development", "sandbox") };
```
and have that box under the hood